### PR TITLE
Introduce game server

### PR DESCRIPTION
**Pull request by smbh**

### CODE REVIEW

File: src/constant.py [hunk 1]
```diff
@@ -3,3 +3,8 @@
 LOBBY_DOMAIN_NAME = '0.0.0.0'
 LOBBY_PORT = 8000
 LOBBY_URL = LOBBY_SCHEME + LOBBY_DOMAIN_NAME + ':' + str(LOBBY_PORT)
+
+GAME_SERVER_SCHEME = 'http://'
+GAME_SERVER_DOMAIN_NAME = '0.0.0.0'
+GAME_SERVER_PORT = 8001
```
Comment (piowag) [inline src/constant.py]: port needs to be added from command line during startup

File: src/game_server_handler.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+class GameServerHandler:
+    def __init__(self, public_uuid):
+        self.public_uuid = public_uuid
+        self.players = dict()
+        self.state = GameState.waiting_for_players
+
+    def add_player_to_game(self, token):
+        if self.state is not GameState.waiting_for_players:
+            return {'msg': "Game already running"}
```
Comment (Donaim) [inline src/game_server_handler.py]: Some kind of "status" would be nice to inform lobby/UI if operation succeeded

Comment (Donaim) [reply src/game_server_handler.py]: So something like
```
{'status': 'ERROR',
 'msg': 'Game already running' }

{'status': 'OK',
 'msg': 'Player added'}
```

Comment (smbh) [reply src/game_server_handler.py]: Will do

**Pull request merged**